02week/pigLatin.js

```diff
@@ -9,12 +9,19 @@ const rl = readline.createInterface({
 
 
 const pigLatin = (word)  => {
-
+  word = word.toLowerCase();
```

Comment:
Nice concise solution.
This will treat words that do not have vowels as if they started with a vowe,